Add ssl default settings in http context. (#13)
chr4/salt-nginx: README.md

Install the `nginx` package from the official PPA.

> Set default ssl settings in the `http` context. This settings can be overwritten in each server if needed. 

Reviewer: ```suggestion
Set default ssl settings in the `http` context. This settings can be overwritten for `server {}` block if needed. 
```

Reviewer: Bit more clear this way, as "Server" can also mean instance, etc.
